arm:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  joint1_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint2_to_joint1
    pid: {p: 120.0, i: 0.01, d: 0.01}

  joint2_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint3_to_joint2
    pid: {p: 120.0, i: 0.01, d: 0.01}

  joint3_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint4_to_joint3
    pid: {p: 120.0, i: 0.01, d: 0.01}

  joint4_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint5_to_joint4
    pid: {p: 120.0, i: 0.01, d: 0.01}

  joint5_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint6_to_joint5
    pid: {p: 120.0, i: 0.01, d: 0.01}

  joint6_effort_controller:
    type: effort_controllers/JointEffortController
    joint: joint6output_to_joint6
    pid: {p: 120.0, i: 0.01, d: 0.01}


